crates/node/core/src/args/network.rs

```diff
@@ -96,6 +97,12 @@ pub struct NetworkArgs {
     #[arg(long)]
     pub max_inbound_peers: Option<usize>,
 
+    /// Max number of seen transactions to remember per peer.
+    ///
+    /// Default is 10 KiB, i.e. 320 transaction hashes.
```

Comment:
```suggestion
    /// Default is 320 transaction hashes.
```

this should be enough context